cypress/support/step_definitions/validations.js

```diff
@@ -49,10 +49,13 @@ Given(
         fireboltItems.forEach((item) => {
           const validationType = item.event ? CONSTANTS.EVENT : CONSTANTS.METHOD;
 
```

Comment:
we cannot keep manage_ in the code. it would be proprietary. That has to be removed

Reply:
updated the code